cppyy uses manylinux (https://github.com/pypa/manylinux) to create the
wheels for the pip packages. Unfortunately, they are not fully compatible with
conda-forge, so we need to build our own; see
https://bitbucket.org/wlav/cppyy/issues/109/manylinux-wheel-is-using-non-manylinux.
diff --git a/cling/setup.py b/cling/setup.py
index b0a4b31..04aaac6 100755
--- a/setup.py
+++ b/setup.py
@@ -26,19 +26,8 @@ with codecs.open(os.path.join(here, 'README.rst'), encoding='utf-8') as f:
 #
 # platform-dependent helpers
 #
-_is_manylinux = None
 def is_manylinux():
-    global _is_manylinux
-    if _is_manylinux is None:
-        _is_manylinux = False
-        try:
-            for line in open('/etc/redhat-release').readlines():
-                if 'CentOS release 5.11' in line:
-                    _is_manylinux = True
-                    break
-        except (OSError, IOError):
-            pass
-    return _is_manylinux
+    return True
 
 def get_build_type():
     if is_manylinux() or 'win32' in sys.platform:
